packages/dmn-js-decision-table/assets/css/dmn-js-decision-table.css

```diff
@@ -317,6 +317,30 @@
   text-overflow: ellipsis;
 }
 
+.dmn-decision-table-container thead {
+  counter-reset: inputs outputs;

```

Comment:
:+1: how we use `counters`

Reply:
I should use that for the When/And/Then clauses too.